mod_reforged/hooks/skills/actives/reload_bolt.nut

```diff
@@ -3,6 +3,7 @@
 	{
 		__original();
 		this.m.IsRemovedAfterBattle = false;
+		this.m.Order -= 100;	// Player expect this perk to come earlier than most other active skills because of muscle memory, so we move it forward a bit
```

Comment:
- Comment should say `skill`, not `perk`.
- After the proposed skill order change, the comment can be updated to say that we set it like this so that it appears immediately after the shoot bolt skill.